CONTRIBUTING.md

```diff
@@ -13,6 +13,49 @@ and let our peers review the code before it gets merged.
 - We try to prefix our commit messages with a word that quickly tells the reader where the change happened. Examples are `editor`, `doc`, `<component-name>`, etc.
 - Changes that touch the core project should be unit-tested. 
 
+### Unity License Setup For Automated Testing
+
+All Pull Request must pass an automated run of all the unit tests before merging. These will fail until your fork of the `VisualPinball.Engine` is configured with Unity's license information.
+Most contributors will be using a Personal license and will need to request a key on behalf of GitHub. Professional license users can gather their key from the [Unity Subscriptions Page](https://id.unity.com/en/subscriptions) and skip to step 8.
+
+1. Create a new branch. We will use the [Unity - Request Activation File](https://github.com/marketplace/actions/unity-request-activation-file) action to request an activation file
+2. Create a file called `.github/workflows/activation.yml` and add the following workflow action defintion.
+```
+name: Acquire activation file
+on: push
+jobs:
+  activation:
+    name: Request manual activation file 
+    runs-on: ubuntu-latest
+    steps:
+      # Request manual activation file
+      - name: Request manual activation file
+        id: getManualLicenseFile
+        uses: game-ci/unity-request-activation-file@v2
+      # Upload artifact (Unity_v20XX.X.XXXX.alf)
+      - name: Expose as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ steps.getManualLicenseFile.outputs.filePath }}
+          path: ${{ steps.getManualLicenseFile.outputs.filePath }}
+```
+3. Commit and Push the new file.
+4. Navigate to the Actions Tab of GitHub.
+5. Once the action has completed download the manual activation file that now appeared as an artifact and extract the `Unity_v20XX.X.XXXX.alf` file from the zip.
+6. Visit (license.unity3d.com) and upload the `Unity_v20XX.X.XXXX.alf` file.
+7. You should now receive your license file (Unity_v20XX.x.ulf) as a download. It's ok if the numbers don't match your Unity version exactly.
+8. Open `Github` > `<Your repository>` > `Settings` > `Secrets`
+- Create the following secrets
+  - `UNITY_EMAIL` - (Add the email address that you use to login to Unity)
+  - `UNITY_PASSWORD` - (Add the password that you use to login to Unity)
+- Personal License
+  - `UNITY_LICENSE` - (Copy the contents of your ulf license file into here)
+- Professional License
+  - `UNITY_SERIAL` - (Add you serial key it should look like XX-XXXX-XXXX-XXXX-XXXX-XXXX)

```

Comment:
After talking with @jsm174, the preferred method would be to actually include `activation.yml` in the repo so the user must only set the correct secrets instead of additionally copying everything over.